packages/create-astro/src/actions/dependencies.ts

```diff
@@ -0,0 +1,44 @@
+import type { Context } from "./context";
+
+import { spinner } from '@astrojs/cli-kit';
+import { title, info } from '../messages.js';
+import { execa } from 'execa';
+
+export async function dependencies(ctx: Pick<Context, 'install'|'yes'|'prompt'|'pkgManager'|'cwd'|'dryRun'>) {
+	let deps = ctx.install ?? ctx.yes;
+	if (deps === undefined) {
+		({ deps } = await ctx.prompt({
+			name: 'deps',
+			type: 'confirm',
+			label: title('deps'),
+			message: `Install dependencies?`,
+			hint: 'recommended',
```

Comment:
Pointing out this is the only prompt with `hint: 'recommended'` rather than `hint: '(recommended)'`, if this wasn't intentional I'd change for consistency's sake:

```suggestion
hint: '(recommended)',
```